### PR TITLE
UF-10716: Upgraded case data to 9.0

### PR DESCRIPTION
**Pull request by MickeWallner**
## Types of changes

- [ ] Bug fix
- [ ] New feature
- [ ] Removed feature
- [ ] Code style update (formatting etc.)
- [x] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] Documentation content changes

## Does this PR introduce a breaking change?
- [ ] Yes (I have stepped the version number accordingly)
- [x] No
      
## Checklist:

- [x] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly (if applicable).
- [x] I have added/updated tests to cover my changes (if applicable).

### CODE REVIEW

**Review by ronnberg — COMMENTED** [review]
Looks ok except the wildcard imports in a couple of files.

File: src/main/java/se/sundsvall/parkingpermit/Constants.java [hunk 1]
```diff
@@ -33,6 +33,8 @@ private Constants() {}
 	public static final VariableValueDto TRUE = new VariableValueDto().type(ValueType.BOOLEAN.getName()).value(true);
 	public static final VariableValueDto FALSE = new VariableValueDto().type(ValueType.BOOLEAN.getName()).value(false);
 
+	//TODO Remove when API is updated
```
Comment (foobar-1337) [inline src/main/java/se/sundsvall/parkingpermit/Constants.java:36]: Maybe be a bit more specific? Are we waiting for a story to be completed? 

I.e: what are supposed to be updated in order for this variable to be removed?

**Comment by foobar-1337** [comment]
Lift to dept44 6.0.3?

**Comment by MickeWallner** [comment]
> Lift to dept44 6.0.3?

Will be done in a seperate PR.

**Review by ronnberg — APPROVED** [review]

**Pull request merged**